lib/libutils/isoc/include/assert.h

```diff
@@ -32,13 +32,17 @@
 void _assert_break(void) __noreturn;
 void _assert_log(const char *expr, const char *file, int line);
 
+#ifdef NDEBUG
+#define assert(expr)
```

Comment:
It has to expand to something, for instance `do { } while (0)` or `((void)0)` as it could otherwise result in warnings for some compiler options.